### PR TITLE
Fee request API updates

### PR DESCRIPTION
**Pull request by Ation**

### CODE REVIEW

File: Blocksettle_proto/Communication/bs_api_server.proto [hunk 1]
```diff
@@ -386,12 +386,13 @@ message Response
...
-        string error_msg      = 2;
-        string product_type   = 3;
-        string fee            = 4;
-        string fee_ccy        = 5;
-        string im             = 6;
+        bool   success           = 1;
+        string error_msg         = 2;
+        string product_type      = 3;
+        string fee               = 4;
+        string dealer_comission  = 5;
```
Comment (sergey-chernikov) [inline Blocksettle_proto/Communication/bs_api_server.proto:393]: should be "commission"

File: Blocksettle_proto/Communication/bs_server.proto [hunk 2]
```diff
@@ -2386,12 +2386,13 @@ message CashResMessage
...
-      bool   success = 2;
-      string error_msg = 3;
-      bs.types.XBTPrice fee = 4;
-      string fee_ccy = 5;
-      bs.types.XBTPrice im = 6;
+      string            product_name      = 1;
+      bool              success           = 2;
+      string            error_msg         = 3;
+      bs.types.XBTPrice fee               = 4;
+      bs.types.XBTPrice dealer_comission  = 5;
```
Comment (sergey-chernikov) [inline Blocksettle_proto/Communication/bs_server.proto:2393]: here, too

File: Blocksettle_proto/Communication/bs_api_server.proto (hunk 1 above)
Comment (Ation) [reply Blocksettle_proto/Communication/bs_api_server.proto]: fixed

File: Blocksettle_proto/Communication/bs_server.proto (hunk 2 above)
Comment (Ation) [reply Blocksettle_proto/Communication/bs_server.proto]: fixed

**Review by goatpig — APPROVED** [review]
tACK

**Pull request merged**